index.html

```diff
@@ -0,0 +1,1928 @@
+                'The Walking Dead World Beyond' Release Date: When Will the
+                Spin-Off Air?
+              </h4>
+            </div>
+          </div>
+          <div class="col-sm-4">
+            <div class="feature1">
+              <div class="position-relative">
+                <img
+                  src="https://d.newsweek.com/en/full/1580819/pfc-glendon-oakley.png"
+                  alt="Wuhan"
+                  width="397"
+                  height="265"
+                  class=""
+                />
+                <div class="p-2 position-absolute fixed-bottom section-thumb">
+                  World
+                </div>
+              </div>
+              <h4 class="section-subtitle">
+                Soldier Commended For Actions During El Paso Walmart Shooting
+                Found Dead
+              </h4>
+            </div>
+          </div>
+          <div class="col-sm-4">
+            <div class="d-flex pb-2 mb-2 border-bottom">
+              <div class="mr-2">
+                <img
+                  src="https://d.newsweek.com/en/full/1580806/money-coronavirus-stimulus-package.jpg"
+                  alt="#"
+                  width="150"
+                  height="95"
+                />
+              </div>
+
+              <h6 class="section-milititle">
+                Restauranteur Delivers Oranges to Hospital Workers Amid
+                Coronavirus Crisis
+              </h6>
+            </div>
+            <div class="d-flex pb-2 mb-2 border-bottom">
+              <div class="mr-2">
+                <img
+                  src="https://d.newsweek.com/en/full/1580806/money-coronavirus-stimulus-package.jpg"
+                  alt="#"
+                  width="150"
+                  height="95"
+                />
+              </div>
+
+              <h6 class="section-milititle">
+                Restauranteur Delivers Oranges to Hospital Workers Amid
+                Coronavirus Crisis
+              </h6>
+            </div>
+            <div class="d-flex pb-2 mb-2 border-bottom">
+              <div class="mr-2">
+                <img
+                  src="https://d.newsweek.com/en/full/1580806/money-coronavirus-stimulus-package.jpg"
+                  alt="#"
+                  width="150"
+                  height="95"
+                />
+              </div>
+
+              <h6 class="section-milititle">
+                Restauranteur Delivers Oranges to Hospital Workers Amid
+                Coronavirus Crisis
+              </h6>
+            </div>
+          </div>
+        </div>
+      </div>
+
+      <div class="subscribe mx-1 px-3 pt-3 pb-5" style="background: white;">
+        <div class="row">
+          <div class="col-lg-8">
+            <h4 class="section-milititle" style="color: #f72210;">SUBSCRIBE</h4>
+            <div
+              class="row sub1 mx-1"
+              style="padding-top: 21px; padding-bottom: 13px;"
```

Comment:
The elements in this section should be centered on the mobile version. Please adjust this.
Yours:
![Screenshot from 2020-04-13 13-08-12](https://user-images.githubusercontent.com/51926937/79146372-f0aabe00-7d87-11ea-841c-8285a3a73f8c.png)

Original:
![Screenshot from 2020-04-13 13-08-31](https://user-images.githubusercontent.com/51926937/79146383-f43e4500-7d87-11ea-95a1-649114f6fd7f.png)